src/debugger/types.js

```diff
@@ -30,13 +30,17 @@ export type PrepackLaunchArguments = {
   exitCallback: () => void,
 };
 
-export type BreakpointArguments = {
-  kind: "breakpoint",
+export type Breakpoint = {
   filePath: string,
   line: number,
   column: number,
 };
 
+export type BreakpointsArguments = {
```

Comment:
You used to support the single breakpoint request format. Now you completely switched to multi-breakpoint requests. 
Do you need to support both? 
My understanding is that, debugger UI will send multi-breakpoint requests during launch, but will use single breakpoint request for later individual breakpoint operations so you need both.

Reply:
The protocol has breakpoint requests always send a list of breakpoints, so even with one breakpoint, the UI will send a list and this change can handle it.